qualtrics-lti/Dockerfile.umich

```diff
@@ -1,8 +1,10 @@
 FROM ghcr.io/longhornopen/qualtrics-lti:20230215
 
-WORKDIR /var/www/html
-# Update lax to none in the session config
-RUN sed -i s/lax/none/ config/session.php
+WORKDIR /var/www/html/
+
+# Add our .htaccess into the one there
+COPY .htaccess.append .
+RUN cat .htaccess.append >> public/.htaccess && rm .htaccess.append
```

Comment:
Nice technique!

Reply:
Yeah, it's not ideal, [I found it here](https://github.com/moby/moby/issues/12193) and seems to be the best way to append right now.